### PR TITLE
Firebase

### PR DESCRIPTION
**Pull request by wscheffer**
This should work fine just can't get the ng-model on the ng-select to work

### CODE REVIEW

File: www/app/page5.controller.js [hunk 1]
```diff
@@ -13,16 +13,19 @@ module.exports = function ($scope) {
...
-  $scope.currentroom = document.domain;
+  //$scope.currentroom = document.domain;
   $scope.rooms = [];
   $scope.messages = [];
   $scope.newMessage = {};
 
   //For localhost testing
   if (document.domain == 'localhost') {
     $scope.currentroom = 'localhost:8080';
+    $scope.rooms.push({
```
Comment (wscheffer) [inline www/app/page5.controller.js:24]: @jannesvh  Did you check that this does not cause "localhost:8080" to be duplicated in $scope.rooms when the firebase add event is triggered ?

File: www/app/page5.controller.js [hunk 2]
```diff
@@ -73,12 +73,13 @@ module.exports = function ($scope) {
     let room = data.val()
 
     //Push room info to rooms array
-    
-    $scope.rooms.push({
-      key: data.key,
-      url: room.url
-    });
-    $scope.$apply();
+    if ($scope.currentroom != room.url) {
```
Comment (wscheffer) [inline www/app/page5.controller.js]: Thanks

**Comment by jaderabbit** [comment]
I'll merge. Need to get travis to pass

**Comment by wscheffer** [comment]
Cool Thanks

**Comment by jaderabbit** [comment]
Problem was this: https://github.com/npm/npm/issues/14042
Downgraded to node 6.5 and it works now

**Pull request merged**